python/lsst/sims/seeingModel/seeingModel.py

```diff
@@ -0,0 +1,172 @@
+from __future__ import division
+from builtins import object
+import os
+import warnings
+import numpy as np
+try:
```

Comment:
I would actually prefer a different behavior where you just ignore `photUtils` completely here. If the user want to use `photUtils` it is better to do it actively, using  `filter_effwavelens` not by some hidden logic here.